spec/imsc-hrm.html

```diff
@@ -684,14 +694,15 @@ <h2>Paint Text</h2>
         </tbody>
       </table>
 
-      <p class='note'>The choice of font by the presentation processor can increase rendering complexity. For instance, a cursive
-      font can generally result in a given character yielding different typographical glyphs depending on context, even if latin
-      script is used.</p>
+      <p class='note'>The choice of font by the presentation processor can increase rendering complexity.
+        For instance, a cursive font might select different glyphs for a given [=grapheme=]
```

Comment:
```suggestion
        For instance, a cursive font might select different glyphs for a given <dfn data-lt="i18n-glossary/#def_grapheme">grapheme</dfn>
```

Reply:
Addressed in ee04753

Reply:
Will change to explicit link as discussed in the TTWG call today.

Reply:
Addressed in a16bbf1

Reply:
Also opened #47 regarding how to present terms and definitions, as a separate issue.